pkg/dns/vips_allocator_test.go

```diff
@@ -380,6 +386,48 @@ var _ = DescribeTable("outboundView",
 			},
 		},
 	}),
+	Entry("two external services with same address and disableHostDNSEntry set to true", outboundViewTestCase{
```

Comment:
Maybe worth adding a test with a single one set to true?

Reply:
And one with different port?

Reply:
added the above test cases ☝️